esmvaltool/recipes/recipe_diurnal_temperature_index_wp7.yml

```diff
@@ -0,0 +1,60 @@
+###############################################################################
+# namelist_diurnal_temperature_index.yml
+#
+# Description
+# Namelist for computing the  diurnal temperature indicator.
+#
+# Authors
+# Alasdair Hunter (BSC, Spain)
+# Nicolau Manubens (BSC, Spain)
+# Louis-Philippe  (BSC, Spain)
+#
+# Project
+# C3S Magic
+#
+#Reference
+#http://secif.ipsl.fr/images/SECIF/documents/Communication/fiche_invulnerable/RC_indicateur_EN.pdf
```

Comment:
Link does not work

Reply:
Thanks, @bouweandela. We've remove this link.